scripts/auxtel/scheduler/enable.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+import asyncio
+
+from lsst.ts.idl.enums.Scheduler import SalIndex
+from lsst.ts.standardscripts.scheduler.enable import Enable
+
+
+class ATSchedulerEnable(Enable):
```

Comment:
Thank you for moving this here.

Now a question on timing: I can have a salobj update ready in a day or two that allows kwargs to amain. Would you be interested in using that (and thus further simplifying these new scripts) or is it better to rely on salobj 7.0.0.